src/components/body/content/categories/CategoriesUser.tsx

```diff
@@ -0,0 +1,25 @@
+import {useAppSelector} from "../../../../store/hooks";
+import IT from "../../../icons/examplImage/IT.svg";
+import Loader from "../../../Loader";
+
+const CategoriesUser = () => {
+  const allCategories = useAppSelector((state) => state.categories.categories);
+  const loading = useAppSelector((state) => state.categories.loading);
+
+  return (
+    <div className="categories-user">
+      {loading && <Loader/>}
+      {allCategories.map(category => category.status === "active" ?
```

Comment:
{allCategories.map(category => category.status === "active" && ...

without null at the end